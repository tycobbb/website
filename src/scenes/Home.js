// @flow
import React, { Component } from 'react'
import { styles } from '../shared'

export class Home extends Component<*> {
  render () {
    return <div {...rules.container}>
      <h1>Hi I'm Ty.</h1>
      <p>I'm a mobile & web developer, and I'm most interested in using technology to tackle existing community/social/civic issues. I also like learning new things, games, and all that jazz.</p>
    </div>
  }
}

const rules = styles.sheet({
  container: {
    '> h1': {
      marginBottom: 20
    }
  }
})
