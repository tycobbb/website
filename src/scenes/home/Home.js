// @flow
import React, { Component } from 'react'
import { stylesheet, mixins } from '../../shared/styles'

export class Home extends Component<*> {
  render () {
    return <div {...rules.home}>
      <h1>Hi, I'm Ty.</h1>
      <p>I'm a mobile & web developer, and I'm most interested in using technology to tackle existing community/social/civic issues. I also like learning new things, games, and all that jazz.</p>
    </div>
  }
}

const rules = stylesheet({
  home: {
    '> h1': {
      ...mixins.header
    }
  }
})
