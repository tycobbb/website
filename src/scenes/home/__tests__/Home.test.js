/* eslint-env jest */
import React from 'react'
import { shallow } from 'enzyme'
import { Home } from '../Home'

describe('#render', () => {
  it('renders properly', () => {
    const subject = shallow(<Home />)
    expect(subject).toMatchSnapshot()
  })
})
