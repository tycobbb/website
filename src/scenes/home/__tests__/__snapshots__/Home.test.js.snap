// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`#render renders properly 1`] = `
<div
  data-css-tsigfw=""
>
  <h1>
    Hi, I'm Ty.
  </h1>
  <p>
    I'm a mobile & web developer, and I'm most interested in using technology to tackle existing community/social/civic issues. I also like learning new things, games, and all that jazz.
  </p>
</div>
`;
